examples/tests/sdk_tests/bucket/rename.test.w

```diff
@@ -0,0 +1,52 @@
+bring cloud;
+bring util;
+bring expect;
+
+let b = new cloud.Bucket();
+
+test "rename()" {
+  let assertThrows = (expected: str, block: (): void) => {
+    let var error = false;
+    try {
+      block();
+    } catch actual {
+      assert(actual.contains(expected));
```

Comment:
why not equals?

Reply:
That was introduced by @Chriscbr in #4697. I think the reasoning is that you are generally matching only a part of an error string (we do the same in wingsdk tests using regex).

Reply:
I see, 
I am not sure if we need this adjustment in all of them, as the errors should be the same across targets (if not containing information from the original error thrown by the target)
can you please open an issue for fixing them one day?

Reply:
you can connect the new issue to that one: https://github.com/winglang/wing/issues/5238, to be done after we'll have an `expect.error` in the expect lib (and avoid duplicated work)